### DISCUSSION TITLE
shorten more questions?

### DISCUSSION THREAD

**Discussion by westnordost** [streetcomplete/StreetComplete#3906] [Ideas]
I am in the park right now and stumbled upon this
![Screenshot_20220322-143423_StreetComplete](https://user-images.githubusercontent.com/4661658/159494010-84d898a0-cbdc-4030-ad8a-bd965a8a1851.jpg)

Do you think it would make sense to remove the mention of the object name also for titles where no string replacement is used?

**Comment by matkoniecz** [comment]
In this case I would consider it fine as is and shortening titles would reduce clarity.

**Reply by westnordost** [reply]
So far there hasn't been a lot of feedback about it, though Flo and Matja seem to be in favour. How would you say does this reduce clarity?

**Reply by matkoniecz** [reply]
Because to understand what is going on you need to read both question and extra explanation, rather than just question.

**Comment by mnalis** [comment]
You mean in translation text, e.g. like

```
Is this covered (protected from rain)?
Picnic table
```

Then yes, that sounds good (I assume it would mark all such translations fuzzy, so translators can update it too). 

If you however mean leaving translation as-is, but removing second line (object type) for some quests, then I would not like that (I like how it is unified now, also e.g. because https://github.com/streetcomplete/StreetComplete/pull/3521#discussion_r830513602)

**Reply by westnordost** [reply]
> Is this covered (protected from rain)?
> Picnic table

Yes
